config/proto_conversions.go

```diff
@@ -151,13 +153,22 @@ func ComponentConfigFromProto(protoConf *pb.ComponentConfig) (*resource.Config,
 		return nil, err
 	}
 
+	level := logging.INFO
+	if protoConf.GetLogConfiguration() != nil {
+		if level, err = logging.LevelFromString(protoConf.GetLogConfiguration().Level); err != nil {
+			// Don't fail configuration due to a malformed log level. Future note, if a logger becomes
+			// available in the method, log a warning in this case.
```

Comment:
[q] what do you mean by `a logger becomes available in this method` ?

Reply:
I think it's worth logging a warning if somehow app sent e.g: `log_level: "not a level"`. But there's nothing to log it to. Though saying this I now remember we have the logger formerly known as `Gloabl`.